### PR TITLE
Add Troi account prerequisites to Readme

### PR DESCRIPTION
**Pull request by benjaminaaron**

### CODE REVIEW

**Review by mpanne — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,12 +1,25 @@
-## Get started
+# 📃⏳ Achill
+Web app to help you fill out Troi. Uses the [Svelte](https://svelte.dev/) framework and the [Troi API](https://v2.troi.dev/).
+
+#### 👉 Also check out our other in-house self-built Slack bot for Troi: [BleibTroy](https://github.com/digitalservice4germany/troi-slack-bot)
```
Comment (frederike-ramin) [inline README.md:4]: Amazing! :D "Bleibtroy". 😂

**Review by frederike-ramin — APPROVED** [review]
Amazing! 💯

File: README.md (hunk 1 above)
Comment (frederike-ramin) [reply README.md:4]: @benjaminaaron To fix the build, do you maybe wanna run `npm run format`?

Comment (benjaminaaron) [reply README.md:5]: Hehe 😄 
Done ✅

**Pull request merged**